cmd/gateway/main.go

```diff
@@ -31,5 +35,10 @@ func main() {
 		),
 	)
 
+	// health endpoints (pprof)
```

Comment:
@wfernandes It would probably be better style wise to just remove `logcache.WithGatewayBlock()` above and have the `gateway.Start()` be non-blocking and then block on pprof listening. This is the way logcache cmd does it. Keep things consistent.